.gitignore

```diff
@@ -19,8 +19,3 @@ build
 node_modules
 package-lock.json
 *.log
-
-# Editor files
-.idea
```

Comment:
You need to add these back, otherwise I will absolutely commit these files on accident.

Reply:
@rosskevin Add them in your `~/.gitignore`.

Reply:
But yeah. I'm adding them back for other people.

Reply:
Glad to hear. When we added these it was discussed and agreed that it's zero overhead for everyone's benefit. Happy to see more examples added as needed.

Reply:
I have added a comment.